### PR TITLE
add analysis-bench to benchmark incremental analysis

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: crates/ra_batch/src/vfs_filter.rs [hunk 1]
```diff
@@ -1,54 +1,54 @@
-use std::path::PathBuf;
```
Comment (bjorn3) [inline crates/ra_batch/src/vfs_filter.rs]: Did you change the kind of newline in this file? According to git every line in this file has changed.

Comment (matklad) [reply crates/ra_batch/src/vfs_filter.rs]: Yeah, it used dos newlines before

**Pull request merged**